_posts/2015-09-24-gsoc-2015-wrapping-up.markdown

```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+title: "Google Summer of Code 2015: Wrapping Up"
+categories:
+- activism
+author: Federico Builes
+published: true
+date: 2015-09-24 15:00:00 -07:00
+---
+
+This year marked our fourth participation in [Google's Summer of
+Code](https://developers.google.com/open-source/gsoc/) (GSoC). In this post we'd like to tell you a bit
+about this year's projects:
+
+## Asset Source Maps ([Andrei Istratii](https://github.com/Andreis13))
+Andrei has added support for several types of [Source
+Maps](https://developer.mozilla.org/en-US/docs/Tools/Debugger/How_to/Use_a_source_map) to
+Sprockets 4. With Source Maps users can see a readable version of their code inside Developer
+Tools (Javascript, CSS, etc.) after it's been minified or compiled. Check out Andrei's
+[sprockets](https://github.com/Andreis13/sprockets-rails/commits/master) and
```

Comment:
Almost all work from Andrei is already merged and we can point to them directly. Let me get the links

https://github.com/rails/sprockets/commits?author=Andreis13
https://github.com/rails/sprockets-rails/commits?author=Andreis13
https://github.com/rails/sass-rails/commits?author=Andreis13